spec/services/finance/ecf/output_calculator_spec.rb

```diff
@@ -1062,5 +1083,123 @@ def setup_statement_three
         end
       end
     end
+
+    context "extended" do
+      let!(:schedule) { create(:ecf_extended_schedule) }
+
+      let!(:extended_participant_declaration) do
+        travel_to first_statement.deadline_date - 1.day do
+          create(:ect_participant_declaration, :extended, cpd_lead_provider:)
+        end
+      end
+
+      it "returns correct bands" do
+        extended_keys = %i[
+          band
+          min
+          max
+
+          previous_extended_1_count
+          extended_1_count
+          extended_1_additions
+          extended_1_subtractions
+
+          previous_extended_2_count
+          extended_2_count
+          extended_2_additions
+          extended_2_subtractions
+
+          previous_extended_3_count
+          extended_3_count
+          extended_3_additions
+          extended_3_subtractions
+        ]
+
+        expected = [
+          {
+            band: :a,
+            min: 1,
+            max: 2,
+
+            previous_extended_1_count: 0,
+            extended_1_additions: 1,
+            extended_1_count: 1,
+            extended_1_subtractions: 0,
+
+            previous_extended_2_count: 0,
+            extended_2_additions: 0,
+            extended_2_count: 0,
+            extended_2_subtractions: 0,
+
+            previous_extended_3_count: 0,
+            extended_3_additions: 0,
+            extended_3_count: 0,
+            extended_3_subtractions: 0,
+          },
+          {
+            band: :b,
+            min: 3,
+            max: 4,
+
+            previous_extended_1_count: 0,
+            extended_1_additions: 0,
+            extended_1_count: 0,
+            extended_1_subtractions: 0,
+
+            previous_extended_2_count: 0,
+            extended_2_additions: 0,
+            extended_2_count: 0,
+            extended_2_subtractions: 0,
+
+            previous_extended_3_count: 0,
+            extended_3_additions: 0,
+            extended_3_count: 0,
+            extended_3_subtractions: 0,
+          },
+          {
+            band: :c,
+            min: 5,
+            max: 6,
+
+            previous_extended_1_count: 0,
+            extended_1_additions: 0,
+            extended_1_count: 0,
+            extended_1_subtractions: 0,
+
+            previous_extended_2_count: 0,
+            extended_2_additions: 0,
+            extended_2_count: 0,
+            extended_2_subtractions: 0,
+
+            previous_extended_3_count: 0,
+            extended_3_additions: 0,
+            extended_3_count: 0,
+            extended_3_subtractions: 0,
+          },
+          {
+            band: :d,
+            min: 7,
+            max: 8,
+
+            previous_extended_1_count: 0,
```

Comment:
maybe we can add some more extended declarations to show a few calculations for other extended declarations rather than 0?

Reply:
Added `extended-2` and `extended-3` declarations.